interop-devnet/docker-compose.yml

```diff
@@ -0,0 +1,343 @@
+# This Compose file is expected to be used with the devnet-up.sh script.
+# The volumes below mount the configs generated by the script into each
+# service.
```

Comment:
is it the case it's expected to be used with `devnet-up`? I don't see any modifications to that script so I wanted to make sure this wasn't a copy-paste error.